test/src/sdk-harness/test_projects/block/mod.rs

```diff
@@ -0,0 +1,109 @@
+use fuels::prelude::*;
+use std::time::{SystemTime, UNIX_EPOCH};
+use tokio::time::{sleep, Duration};
+
+abigen!(
+    BlockTestContract,
+    "test_projects/block/out/debug/block-abi.json"
+);
+
+async fn get_block_instance() -> (BlockTestContract, ContractId) {
+    let wallet = launch_provider_and_get_wallet().await;
+    let id = Contract::deploy(
+        "test_projects/block/out/debug/block.bin",
+        &wallet,
+        TxParameters::default(),
+        StorageConfiguration::with_storage_path(Some(
+            "test_projects/block/out/debug/block-storage_slots.json".to_string(),
+        )),
+    )
+    .await
+    .unwrap();
+    let instance = BlockTestContractBuilder::new(id.to_string(), wallet).build();
+
+    (instance, id.into())
+}
+
+#[tokio::test]
+async fn can_get_block_height() {
+    let (instance, _id) = get_block_instance().await;
+    let block_0 = instance.get_block_height().call().await.unwrap();
+    let block_1 = instance.get_block_height().call().await.unwrap();
+    let block_2 = instance.get_block_height().call().await.unwrap();
+
+    // Consecutive blocks
+    assert_eq!(block_1.value, block_0.value + 1);
```

Comment:
This seems brittle; it will break when there is more than 1 tx per block right?
That said, it's probably good enough for now and can be updated when it does break.

Reply:
Yes you're right. I changed the checks to be a bit more flexible and more future proof.